src/main/java/pucsp/mycash/model/Transaction.java

```diff
@@ -0,0 +1,27 @@
+package pucsp.mycash.model;
+
+import jakarta.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name="transaction")
+public class Transaction {
+    @Id
+    @GeneratedValue(strategy=GenerationType.AUTO)
+    private Long id;
+    @Temporal(TemporalType.DATE)
+    @Column(nullable = false)
+    private Date date;
+    private float value;
+    @Column(nullable = false)
+    private String description;
+    @ManyToOne
+    @JoinColumn(name = "category_id")
+    private TransactionCategory categoryId;
```

Comment:
`@JsonBackReference`
```suggestion
    private TransactionCategory category;
```